membership-attribute-service/test/controllers/AttributeControllerTest.scala

```diff
@@ -462,13 +479,44 @@ class AttributeControllerTest extends Specification with AfterAll with Idiomatic
       verifyIdentityHeadersSet(result, userWithNewspaperUserId)
 
     }
+
+    "return the correct feast attributes for newspaper plus subscribers" in {
+      val req = FakeRequest().withCookies(newspaperPlusCookie)
+      val result = controller.attributes(req)
+
+      status(result) shouldEqual OK
+      val jsonBody = contentAsJson(result)
+      println(Json.prettyPrint(jsonBody))
+      jsonBody shouldEqual
+        Json.parse(s"""
+                      |{
+                      |  "userId": "$userWithNewspaperPlusUserId",
+                      |  "digitalSubscriptionExpiryDate":"${dateTimeInTheFuture.toLocalDate}",
```

Comment:
this test is the same as the old "paper" one above, apart from this additional line